.github/workflows/linters.yml

```diff
@@ -0,0 +1,33 @@
+name: Check Frontend Linters
+
+on:
+  pull_request:
+    paths:
+      - "UT4MasterServer.Web/**"
+      - ".github/workflows/linters.yml"
+      # ignored paths
+      - "!*.md"
+  push:
+    paths:
+      - "UT4MasterServer.Web/**"
+      - ".github/workflows/linters.yml"
+      # ignored paths
+      - "!*.md"
+
+env:
+  WEB_DIR: UT4MasterServer.Web
+
+jobs:
+  linters:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - run: npm install --ignore-scripts
+        working-directory: ${{ env.WEB_DIR }}
+      - run: npm run lint
+        working-directory: ${{ env.WEB_DIR }}
+      - run: npm run format-check
```

Comment:
This isn't strictly necessary because I have prettier write and eslint fix running as a commit hook, but I wouldn't be against making the github action do prettier write and lint fix and removing the husky hook

Reply:
I won't recommend making commits fixes from auto-linters directly from GH Actions.
They can break your code, for example like auto fixes for `prefer-template` rule (more than 2 spaces, sometimes no spaces)